ignite/cmd/scaffold_react.go

```diff
@@ -0,0 +1,36 @@
+package ignitecmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/services/scaffolder"
+)
+
+// NewScaffoldVue scaffolds a Vue.js app for a chain.
+func NewScaffoldReact() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "react",
+		Short:   "Generate ReactJS web app template",
```

Comment:
The React framework describes itself as "React" in their web, maybe we should use the same name when referring to React apps instead of "ReactJS".

Reply:
I've seen both used but let's go with React